community/open-meetings.mdx

```diff
@@ -58,6 +58,14 @@ These are dedicated sync meetings for specific products, as well as open plannin
              }
 />
 
+<MeetingInfo title="[Portal] Portal Open Meeting"
+             schedule="Every Thursday effective 4. Jul 2024 until 31. Dec 2024 from 01:30 pm to 02:00 pm CEST"
+             description="Coordination of feature refinement and development for portal."
```

Comment:
could you please change to Portal product not just portal.

Reply:
adapted. thx